convert2rhel/actions/report.py

```diff
@@ -75,9 +76,13 @@ def summary_as_json(results, json_file=CONVERT2RHEL_JSON_RESULTS):
     * The results are modified so that status codes use their symbolic names
       instead of the numeric values.
     """
+    # Collect the highest report level to use as status key
+    highest_level = find_highest_report_level(results)
+
     # Use an envelope so we can add other, non-result info if necessary.
     envelope = {
         "format_version": "1.0",
```

Comment:
I'm wondering if the version shouldn't be increased when there is a change in format

Reply:
Correct, i'll add this